crates/loam-cli/README.md

```diff
@@ -0,0 +1,30 @@
+# loam-cli
+
+Build smart contracts authored with Loam SDK, manage smart contract dependencies from a frontend, initialize new loam projects.
+
+Loam CLI comes with three main commands:
+
+* `loam init` - Generates a [Loam frontend](https://github.com/loambuild/template?tab=readme-ov-file) that includes an `environments.toml` file describing the network settings, accounts, and contracts for each environment your team builds against.
+* `loam build` - Two build processes in one:
+  * Build smart contracts. Essentially, this is a wrapper around `cargo build` that will additionally traverse dependencies and find any with a `package.metadata.loam.contract = true` setting in their Cargo.toml, and ensure that they all get built in the correct order. You can use `loam build` to build non-Loam-SDK projects and it will Just Work.
```

Comment:
```suggestion
  * Build smart contracts. Essentially, this is a wrapper around `cargo build` that will additionally traverse dependencies and find any with a `package.metadata.loam.contract = true` setting in their Cargo.toml, and ensure that they all get built in the correct order. You can use `loam build` to build any soroban projects with the contract setting in their `Cargo.toml` and it will Just Work.
```

Reply:
@willemneal I purposely meant that you can, in fact, use this with projects that do _not_ have the `package.metadata.loam.contract = true` setting, too. I tried and it works! Because it's just wrapping `cargo build`, with the extra dependency-traversal stuff. I found this clarifying, personally, but maybe it's not that helpful to others. ¯\\\_(ツ)_/¯

Reply:
It's more than just cargo build. It sets the crate type and so not generally meant to be for any crate.